src/DynamoCoreWpf/Views/Core/ConnectorPinView.xaml.cs

```diff
@@ -34,6 +34,8 @@ public partial class ConnectorPinView : IViewModelView<ConnectorPinViewModel>
 
         public ConnectorPinView()
         {
+            Resources.MergedDictionaries.Add(SharedDictionaryManager.DynamoConvertersDictionary);
```

Comment:
Hi @SHKnudsen Can you remind me what is this for? I thought at some point we conclude this does not work? Or maybe I misunderstood

Reply:
Need to load the converters to be able to use `InverseBoolToVisibilityCollapsedConverter` so that we can collapse the pins
https://github.com/DynamoDS/Dynamo/pull/12070/files#diff-bb986eed1172de44f4bd46182f227792c03db1a77bbb4d76a206b7f8c0c126e0R15

Reply:
Can you put this as a comment? Once that is done, this is good to go, thanks for doing it